.storybook/pages/CadModal/ModalFirstStep.tsx

```diff
@@ -0,0 +1,66 @@
+import React from 'react';
+
+import { Button, ButtonGroup, Icon, Modal, Heading } from '../../../src';
+
+import { RadioField } from '../../../src/upcoming-components/RadioField/RadioField';
+
+export const ModalFirstStep = () => (
+  <Modal
+    onClose={
+      () => {} /* eslint-disable-line @typescript-eslint/no-empty-function */
+    }
+    open
+  >
+    <Modal.Header>
+      <Modal.Title className="u-margin-bottom-md">
+        Before we start, tell us your planning needs
+      </Modal.Title>
+      <Heading as="h3" size="title-md">
+        We’ll need to know how to set up your plan. Can you tell us how you’ll
+        use the planner?
+      </Heading>
+    </Modal.Header>
+    <Modal.Body>
+      <RadioField label="Content pieces go here">
+        <RadioField.Item
+          name="course-planner"
+          text="I need my own planner. No additional teachers will need to create a plan for this course."
+          value="radio1"
+        />
+        <RadioField.Item
+          name="course-planner"
+          text="I need my own plans, but other teachers may need their own plans for this course too."
+          value="radio2"
+        />
+        <RadioField.Item
+          name="course-planner"
+          text="I’ll review and use a plan set by another teacher."
+          value="radio3"
+        />
+      </RadioField>
+    </Modal.Body>
+    <Modal.Footer className="u-display-flex u-justify-content-space-between">
+      <Modal.Stepper activeStep={1} totalSteps={3} />
```

Comment:
I think your stepper needs a little bottom margin or something.
<img width="1663" alt="first step story in storybook. there is no space between the stepper icons and the buttons in the footer" src="https://user-images.githubusercontent.com/7761701/173100404-27a773c1-daab-4cc8-94cf-8ac3e90d1f30.png">

Reply:
Can I bug you for your opinion on this? I think the mocks wanted to just have the two optional components (like a right slot and a left slot) but I felt like that was too strict and made it open for any `React.children`. 

In the use case here I passed in the stepper and the buttongroup and basically want to do:
```css
display: flex;
align-items: center;
justify-content: space-between;
```
but pages are not supposed to have any custom page level css.

What are your thoughts?

Reply:
Btw on a side note, this was caused by not importing the utility classes, reverted that commit to import them to fix styling